_pubs/charting-eda.md

```diff
@@ -0,0 +1,26 @@
+---
+stub: true
+title: "Charting EDA: Characterizing Interactive Visualization Use in Computational Notebooks with a Mixed-Methods Formalism"
+authors:
+  - key: dwootton
+  - key: amyfox
+  - key: peck 
+  - key: arvindsatya
+venue: infovis
+year: 2024
+date: 2024-10-13
```

Comment:
```suggestion
venue: vis-full
year: 2025
date: 2025-01-01
```

So, VIS full papers (`vis-full` under `venues.yml` rather than the old `infovis` conference) are a weird beast: they appear in a special issue of TVCG that is published in January of the subsequent year.